pareto/utilities/get_data.py

```diff
@@ -54,25 +236,34 @@ def _read_data(_fname, _set_list, _parameter_list):
 
     _df_parameters = pd.read_excel(
         _fname,
-        sheet_name=_parameter_list,
+        sheet_name=None,  # read all tabs
         header=1,
         index_col=None,
         usecols=None,
         squeeze=True,
         keep_default_na=False,
     )
+
+    # Remove tabs that are not specified by user and are not valid PARETO inputs
+    _df_parameters = {
+        key: value
+        for key, value in _df_parameters.items()
+        if key in valid_parameter_tab_names
+    }
+
+    # Cleaning Parameters.
     # A parameter can be defined in column format or table format.
     # Detect if columns which will be used to reshape the dataframe by defining
     # what columns are Sets or generic words
-    # If _set_list is empty, it is assummed that a parameter is column format is being read.
-    # and _set_list is created based on the DataFrame column names, except for the last name,
+    # If valid_set_tab_names is empty, it is assumed that a parameter in column format is being read.
+    # and valid_set_tab_names is created based on the DataFrame column names, except for the last name,
     # which is used as the data column name.
```

Comment:
The way the code is written, it is impossible for `valid_set_tab_names` to be empty. Can you please rewrite these comments to be a little clearer about what is going on here?

Reply:
Good observation. Theses changes were originally to update comments to match the updates variables used in the function. Looking at it again, I think this section again, I think it may actually be problematic with the new set-up of get_data. I will remove and test. 
To give a short description of this section of code - it allowed users to pass in dataset with no sets and parameters in a column-only format. It would then assume the column names were valid sets. We don't have any datasets that follow this format and I wasn't even aware that this was a feature of get_data(). IT would also take some effort to make it work with the new get_data, so I'm opting to remove.